src/components/AddGrid.vue

```diff
@@ -0,0 +1,105 @@
+<template>
+    <div>
```

Comment:
component's root element should have a class with the components name:
```html
<div class="component_name_here">
```

Reply:
Included name  resolved